Make recommendation to not CID for NAT stronger (#306)
quicwg/ops-drafts: draft-ietf-quic-manageability.md

The QUIC header optionally contains a connection ID which can be used as
additional entropy beyond the 5-tuple, if needed. The QUIC handshake needs
to be observed in order to understand whether the connection ID is present and

> what length it has. However, connection IDs may be renegotiated

Author: ```suggestion
what length it has. However, connection IDs may be renegotiated during
```

> If a NAT hits an operational limit, it is recommended to rather drop the

Reviewer: ```suggestion
If a NAT hits an operational limit, it is recommended to drop the
```

Author: This is addressed in another PR.

initial packets of a flow (see also {{sec-filtering}}),
which potentially triggers a fallback to TCP. Use of the connection ID to

> muliplex mulitple connections on the same IP address/port pair is not a

Reviewer 2: ```suggestion
multiplex multiple connections on the same IP address/port pair is not a
```
